client/src/components/layouts/DashboardSidebarData.js

```diff
@@ -46,6 +48,16 @@ const DashboardSidebarData = ({setLogoutAlert}) => {
       link: '/team',
       name: 'TEAM',
     },
+    {
+      title: 'Achievements',
+      icon: <BsFillAwardFill />,
+      link: '/achievement',
+    },
+    {
+      title: 'Settings',
+      icon: <BsGearFill />,
+      link: '/dashboard/settings',
+    },
```

Comment:
I don't think we are implementing `setting` page for Researchify, you might need to remove this part, as well as the import of the gear icon.